render/detailed_node_test.go

```diff
@@ -1,4 +1,4 @@
-package report_test
+package render
```

Comment:
Can we do that in this PR instead? I'm happy with larger PRs if it means we avoid creating future work or technical debt.